lib/ansible/module_utils/network/meraki/meraki.py

```diff
@@ -127,6 +130,26 @@ def define_protocol(self):
         else:
             self.params['protocol'] = 'http'
 
+    def sanitize_keys(self, data):
+        if isinstance(data, dict):
+            items = {}
+            for k, v in data.items():
+                try:
+                    new = { self.key_map[k]: data[k] }
+                    items[self.key_map[k]] = self.sanitize_keys(data[k])
+                except KeyError:
+                    snake_k = re.sub('([a-z0-9])([A-Z])', r'\1_\2', k).lower()
+                    new = { snake_k: data[k] }
+                    items[snake_k] = self.sanitize_keys(data[k])
+            return items
+        elif isinstance(data, list):
+            items = []
```

Comment:
Why not use a list comprehension here? It's probably more efficient:
`return [self.sanitize_keys(i) for i in data]`

Reply:
Because I’ve never used them. But I’ll research them a bit tonight.

Reply:
I haven't made this change yet, but I will.